src/Sign.Cli/PACKAGE.md

```diff
@@ -0,0 +1,25 @@
+## About
+
+Sign CLI is a .NET tool that provides digital signing for .NET assemblies, packages, and other files.
+
+The tool signs files inside-out, starting with the most nested files and then the outer files, ensuring everything is signed in the correct order.
+
+## Usage
+
+- For help with...
+    - Azure Key Vault:  `sign code azure-key-vault --help` 
+    - Trusted Signing:  `sign code trusted-signing --help` 
+    - local signing:  `sign code certificate-store --help` 
```

Comment:
```suggestion
    - Local signing:  `sign code certificate-store --help` 
```

Reply:
"Local signing" is not a proper name like Azure Key Vault and Trusted Signing.